apiserver/facades/agent/uniter/service.go

```diff
@@ -146,8 +146,9 @@ type StatusService interface {
 	// GetApplicationAndUnitStatusesForUnitWithLeader returns the display status
 	// of the application the specified unit belongs to, and the workload statuses
 	// of all the units that belong to that application, indexed by unit name.
-	// If no application is found for the unit name, an error satisfying
-	// [applicationerrors.ApplicationNotFound] is returned
+	// If the specified unit is not the leader of it's application and error satisfying
+	// [statuserrors.UnitNotLeader] is returned. If no application is found for the
```

Comment:
nit: new line between errors

Reply:
I love how we're trying to format javaesque code comments because no tooling exists for this trash.

Reply:
Deleted error spec at request